ai-chat-assistant/README.md

```diff
@@ -7,6 +7,8 @@ categories: ["AI"]
 
 # AI Chat Assistant
 
+<a href="https://app.autokitteh.cloud/template?name=ai-chat-assistant"><img src="../assets/autokitteh-badge.svg" alt="Start with AutoKitteh" width="160" height="20"/></a>
```

Comment:
this does not render. the asset is missing.

Reply:
Fixed